core/services/ocr2/plugins/ccip/internal/rpclib/evm.go

```diff
@@ -35,16 +36,21 @@ const (
 	// 2. 20/4 = 5
 	// 3. 5/4  = 1
 	DefaultRpcBatchBackOffMultiplier = 5
+
+	// DefaultParallelRpcCallsLimit defines the default maximum number of individual in-parallel rpc calls.
+	DefaultParallelRpcCallsLimit = 10
```

Comment:
```suggestion
	// DefaultMaxParallelRpcCalls defines the default maximum number of RPC calls that can be made in parallel.
	DefaultMaxParallelRpcCalls = 10
```

Name wise this seems better, since it is a max. Also, does this get overridden somewhere (i.e via toml config / onchain config / etc.)? If not maybe we can drop the `Default` from all these constants.

Reply:
We should allow overriding this via jobspec config soon.

Reply:
Renamed.